common/source/docs/common-omnibusnanov6.rst

```diff
@@ -107,6 +140,22 @@ Default pin values:
 Optionally add voltage and / or current monitoring using BLHeli_32-capable ESCs. See instructions :ref:`here <common-dshot>` for setting up BLHeli_32 ESC telemetry.
 
 
+V6.x revision
+-------------
+
+V6.x revision has an additional ADC available for external current sensor hardware / PDB connection in 4in1 socket / respective pads.
+
+:ref:`BATT_CURR_PIN <BATT_CURR_PIN>` = 11
+
+:ref:`BATT_CURR_SCALE <BATT_CURR_SCALE>` = 18.2
```

Comment:
this depends entirely on what type of current sensor is used externally, could be any value...probably should not have been in HWDEF

Reply:
while it might have been a copy/paste slip that brought it into the hwdef, imho it's good to have some starting point other than 0. added a note adjust as required.